modules/dagger/arrow-dagger/src/test/kotlin/arrow/dagger/instances/tests/instances.kt

```diff
@@ -1,9 +1,48 @@
 package arrow.dagger.instances.tests
 
-import arrow.core.*
-import arrow.dagger.instances.*
-import arrow.data.*
-import arrow.typeclasses.*
+import arrow.core.EitherPartialOf
+import arrow.core.ForEval
+import arrow.core.ForFunction0
+import arrow.core.ForId
+import arrow.core.ForOption
+import arrow.core.ForTry
+import arrow.core.Function1PartialOf
+import arrow.dagger.instances.ArrowInstances
+import arrow.dagger.instances.CoproductInstances
+import arrow.dagger.instances.EitherInstances
+import arrow.dagger.instances.EitherTInstances
+import arrow.dagger.instances.Function1Instances
+import arrow.dagger.instances.KleisliInstances
+import arrow.dagger.instances.MapKInstances
+import arrow.dagger.instances.OptionTInstances
+import arrow.dagger.instances.SortedMapKInstances
+import arrow.dagger.instances.StateTInstances
+import arrow.data.CoproductPartialOf
+import arrow.data.EitherTPartialOf
+import arrow.data.ForListK
+import arrow.data.ForNonEmptyList
+import arrow.data.ForSequenceK
+import arrow.data.ForSetK
+import arrow.data.KleisliPartialOf
+import arrow.data.MapKPartialOf
+import arrow.data.OptionTPartialOf
+import arrow.data.SortedMapKPartialOf
+import arrow.data.StateTPartialOf
+import arrow.typeclasses.Applicative
+import arrow.typeclasses.ApplicativeError
+import arrow.typeclasses.Bimonad
+import arrow.typeclasses.Comonad
+import arrow.typeclasses.Eq
+import arrow.typeclasses.Foldable
+import arrow.typeclasses.Functor
+import arrow.typeclasses.Monad
+import arrow.typeclasses.MonadError
+import arrow.typeclasses.Monoid
+import arrow.typeclasses.MonoidK
+import arrow.typeclasses.Order
+import arrow.typeclasses.Semigroup
+import arrow.typeclasses.SemigroupK
+import arrow.typeclasses.Traverse
```

Comment:
Configure your imports to fold after a 5+ of them. This becomes unmaintainable after a while.

Reply:
Done! Fixed all over the place now.